.travis.yml

```diff
@@ -110,7 +110,7 @@ install:
   - "VERBOSE=1 make|grep -v -F '\\-\\- Installing:'"
   - "make install 2>&1|grep -v -F 'Installing:'"
   - "cd $TRAVIS_BUILD_DIR"
-  - "python setup.py install --user"
+  - "python setup.py install ${PIP_USER}"
```

Comment:
Use of the environment variable named `PIP_USER` with `python setup.py install` is confusing since `pip` is not being used here AND `$PIP_USER` is meaningful for the `pip` command.

For example, if this line were `pip install .`, the value of `PIP_USER` can be anything that resolves to true. e.g. `PIP_USER=1 pip install .` is equivalent to `pip install --user`.

My recommendation is to either install with `pip install .` for both platforms, specifying `PIP_USER=1` only in linux (or wherever else required) OR leave it as-is, renaming `PIP_USER` to something like `USER_OPTIONS` with the value `--user` (and any additional options specifically required by the environment).